src/core/Formatter.js

```diff
@@ -165,7 +165,7 @@ export default class Formatter {
     }
 
     formatWithSpaceAfter(token, query) {
-        return _.trimEnd(query) + token.value + " ";
+        return this.formatWithoutSpaces(token, _.trimEnd(query)) + " ";
```

Comment:
Why did we change this? If we want to use `formatWithoutSpaces` to reduce duplication then we could use it like this: 

```
return this. formatWithoutSpaces(token, query) + " ";
```

 But I think it was easier to understand `formatWithSpaceAfter` logic when there was a bit of duplication in the code.